.github/workflows/python-release-tests.yml

```diff
@@ -0,0 +1,87 @@
+name: Test Anemoi Packages for Release
+
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+
+jobs:
+  quality:
+    uses: ecmwf/reusable-workflows/.github/workflows/qa-precommit-run.yml@v2
+    with:
+      skip-hooks: "no-commit-to-branch"
+
+  checks:
```

Comment:
I would rename this to `pytest` instead of `checks`. This name shows up in the list of actions that are running under the PR, it makes it clearer what's happening.

And the same in the PR workflow.